app/javascript/components/analytics/widgets/MonitoreesByDateOfExposure.js

```diff
@@ -72,34 +72,53 @@ class MonitoreesByDateOfExposure extends React.Component {
                 </Form.Control>
               </Form.Group>
             </Form.Row>
-            <ResponsiveContainer width="100%" height={400}>
-              <BarChart
-                width={500}
-                height={300}
-                data={this.state.lastExposureDateDate}
-                margin={{
-                  top: 20,
-                  right: 30,
-                  left: 20,
-                  bottom: 5,
-                }}>
-                <CartesianGrid strokeDasharray="3 3" />
-                <XAxis dataKey="name" />
-                <YAxis />
-                <Tooltip />
-                <Legend />
-                <Bar dataKey="High" stackId="a" fill="#FA897B" />
-                <Bar dataKey="Medium" stackId="a" fill="#FFDD94" />
-                <Bar dataKey="Low" stackId="a" fill="#D0E6A5" />
-                <Bar dataKey="No Identified Risk" stackId="a" fill="#333" />
-                <Bar dataKey="Missing" stackId="a" fill="#BABEC4" />
-              </BarChart>
-            </ResponsiveContainer>
+            <Row className="mx-2 mt-2 px-0">
```

Comment:
This is kind of nitpicky but these two columns are almost completely identical and could probably be rendered with a helper function passing in arguments for dataKey, Fill, and the data index.